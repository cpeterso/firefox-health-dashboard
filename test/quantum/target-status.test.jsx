import React from 'react';
import renderer from 'react-test-renderer';
import TargetStatus from '../../src/quantum/target-status';

it('renders correctly', () => {
  const tree = renderer.create(<TargetStatus notes={{ status: 1 }} />).toJSON();

  expect(tree).toMatchSnapshot();
});
