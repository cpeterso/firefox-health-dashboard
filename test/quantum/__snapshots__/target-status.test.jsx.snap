// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="criteria-widget status-unknown widget-status-all narrow-content"
>
  <header
    className="sides-padding"
  >
    <div>
      <h3>
        <span
          dangerouslySetInnerHTML={
            Object {
              "__html": "Risk/Target Status Summary",
            }
          }
        />
      </h3>
      <span
        className="sides-padding"
      />
    </div>
  </header>
  <div
    className="widget-content"
  >
    <div
      className="widget-entry"
    />
    <div
      className="widget-target"
    >
      Target: 
      <span
        dangerouslySetInnerHTML={
          Object {
            "__html": "Be <em>on track</em> to be <em>within target</em>",
          }
        }
      />
    </div>
  </div>
</div>
`;
